.editorconfig

```diff
@@ -0,0 +1,42 @@
+# docs: https://editorconfig.org
+# please make sure to keep this file in sync between repositories
+# to main file should be edited in `swr-prettier-config` and copied where needed
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = tab
```

Comment:
why ist default indent-style `tab` when all others are `space`?

Reply:
JS and others are all tabs, only some space exclusions